src/models/HybridSystem.jl

```diff
@@ -118,12 +118,19 @@ function HybridSystem(::Nothing)
 end
 
 IS.get_name(value::HybridSystem) = value.name
-function set_unit_system!(value::HybridSystem, settings::SystemUnitsSettings)
-    value.internal.units_info = settings
-    value.thermal_unit.internal.units_info = settings
-    value.electric_load.internal.units_info = settings
-    value.storage.internal.units_info = settings
-    value.renewable_unit.internal.units_info = settings
+
+function get_component(value::HybridSystem)
+    components =
+        [value.thermal_unit, value.electric_load, value.storage, value.renewable_unit]
+    filter!(x -> !isnothing(x), components)
+    return components
+end
```

Comment:
Did you intend to include this function?

Reply:
should this be `get_components` ?

Reply:
@daniel-thom we shouldn't export this one, we have specific ones for each component.